### ISSUE TITLE
Static method voor maken van voters

### ISSUE THREAD

**Issue by PiotrBenedysiuk** [jhoorneman/bestRouteFinder#1]
https://github.com/jhoorneman/bestRouteFinder/blob/08f06223361be04a888e51da478d938e4605a2f8/voter.py#L7

Dit moet een static method zijn, netter is wel om het geen onderdeel te maken van voter. Bovendien, je kan @dataclass gebruiken.

Andere zaak: als je de typing module gebruikt dan kan je ook aangeven wat er in de (bijvoorbeeld) lists zit, zoals typing.List[int].

**Comment by jhoorneman** [comment]
Ja, die method was gemaakt toen voter nog de enige class was. Het plan is nu het uitlezen van de google form te verplaatsen naar Election.

Typing module ken ik idd. We hebben die niet gebruikt zodat je de hele code kunt runnen zonder dat je modules nodig hebt, maar misschien komen we daar nog op terug.

**Comment by jhoorneman** [comment]
DataClass kende ik nog niet trouwens, maar ziet er wel nice uit.

**Comment by PiotrBenedysiuk** [comment]
typing zit ingebouwd in python tegenwoordig

**Comment by jhoorneman** [comment]
Oh ik zie het ja. Dataclass ook. Dat is nice! Dan gaan we die wel gebruiken.
